sbnobj/Common/POTAccounting/BNBSpillInfo.h

```diff
@@ -0,0 +1,63 @@
+#ifndef sbncode_BNBSpillInfo_HH
+#define sbncode_BNBSpillInfo_HH
+
+#include <vector>
+
+namespace sbn {
+  class BNBSpillInfo {
+  public:
+    //typedef enum triggertype { kBNB, kEXT, kUnbiased, kNUMI } TriggerType; //!< Which beamline are we looking at
+    
+    unsigned long int spill_time_s; //!< The IFDB Beam Spill Time, unit sec
+    unsigned long int spill_time_ns; //!< The IFDB Beam Spill Time, unit nsec
```

Comment:
Is there a spill ID in the database? if there is any time of single key column in that DB, it may be useful to store it here for effective lookup.

Reply:
sadly there is not, that would make things MUCH easier. In fact, many devices actually report different times and we have to adjust those times or select the ones that are closest in time.